libs/blocks/text/text.js

```diff
@@ -50,6 +50,11 @@ export default function init(el) {
     const elAction = el.querySelector('.action-area');
     if (elAction) elAction.classList.add('body-s');
   }
+  if (el.classList.contains('link-farm')) {
+    el.querySelectorAll('[id^="no-heading"]').forEach(elem => {
+      elem.innerHTML = '';
+    });
+  }
```

Comment:
Does the `no-heading` selector consider localization? The final doc example might just need a `#_dns` in the label. Just a thought